karma.conf.js

```diff
@@ -1,24 +1,48 @@
 module.exports = function(config) {
+    
+    var saucelabsLaunchers = {
+        sl_ie_9: {
+            base: 'SauceLabs',
+            browserName: 'internet explorer',
+            platform: 'Windows 7',
+            version: '9'
+        },
+        sl_chrome: {
+            base: 'SauceLabs',
+            browserName: 'chrome',
+            platform: 'Windows 10',
+            version: 'latest'
+        },
+    };
+    
     config.set({
         files: [
-            {pattern: 'test/*Test.js'},
-            {pattern: 'test/**/*Test.js'},
-        ],
-        exclude: [
-            'test/plugTest.js'
+            {pattern: 'test/karmaWebpackTestEntry.js'}
         ],
```

Comment:
I am waiting for karma webpack plugin, that allows you do configure file pattern and generates a single webpack bundle.